### DISCUSSION TITLE
Just for Fun - generative AI QR Codes

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#407] [General]
These are all the !RESET command, that clears Labs settings.

Which style is your favorite?
![ResetBear](https://github.com/gopro/labs/assets/10856084/9da0aff3-85cd-4775-8105-bbc92f0bd270)
![ResetCircuit](https://github.com/gopro/labs/assets/10856084/1c983fc6-3d3e-4dae-89dc-89842a15c661)
![ResetCity](https://github.com/gopro/labs/assets/10856084/51e299e0-62f4-4469-b6be-59ebb886e36d)
![ResetElements](https://github.com/gopro/labs/assets/10856084/75eeea89-1c0c-4e60-8494-894a05ef1d8f)
![ResetSnowyTown](https://github.com/gopro/labs/assets/10856084/b10b5615-ce40-4efc-baaa-af69b8d6ade2)

Same are more readable that others, but all can work in my tests.

**Comment by DemonViewLLC** [comment]
The 1st one and the last one.  They really did turn out quite well and are easy to see with the human eye (not sure about the A.I. Eye...).
